tox.ini

```diff
@@ -75,3 +69,16 @@ commands_pre =
 commands =
     isort -rc .
     black .
+
+[testenv:spark]
```

Comment:
Can you leave these near the top? Want to group tests vs. style checks/tooling/etc.